pymatgen/io/qchem/outputs.py

```diff
@@ -1007,8 +1059,86 @@ def _read_single_point_data(self):
             # Two lines will match the above; we want final calculation
             self.data['final_energy'] = float(temp_dict.get('final_energy')[-1][0])
 
-    def _read_force_data(self):
```

Comment:
No need to delete _read_force_data, is there?

Reply:
...I don't remember deleting that. Sorry about that; guess I got trigger-happy?